src/pages/ScreenRouting/MediaScreenRouter.tsx

```diff
@@ -8,25 +8,26 @@ import useMedia from '#src/hooks/useMedia';
 import Loading from '#src/pages/Loading/Loading';
 import ErrorPage from '#components/ErrorPage/ErrorPage';
 import type { PlaylistItem } from '#types/playlist';
-import { isEpisode, isSeriesPlaceholder } from '#src/utils/media';
+import { isEpisode, isDeprecatedSeriesFlow } from '#src/utils/media';
 import MediaMovie from '#src/pages/ScreenRouting/mediaScreens/MediaMovie/MediaMovie';
 import MediaSeries from '#src/pages/ScreenRouting/mediaScreens/MediaSeries/MediaSeries';
 import MediaSeriesEpisode from '#src/pages/ScreenRouting/mediaScreens/MediaSeriesEpisode/MediaSeriesEpisode';
 import MediaLiveChannel from '#src/pages/ScreenRouting/mediaScreens/MediaLiveChannel/MediaLiveChannel';
 import { ScreenMap } from '#src/pages/ScreenRouting/ScreenMap';
+import { CONTENT_TYPE } from '#src/config';
 
 export const mediaScreenMap = new ScreenMap<PlaylistItem>();
 
-// register media screens
-mediaScreenMap.registerByContentType(MediaSeries, 'series');
-mediaScreenMap.registerByContentType(MediaSeriesEpisode, 'episode');
-mediaScreenMap.registerByContentType(MediaLiveChannel, 'livechannel');
-mediaScreenMap.registerByContentType(MediaStaticPage, 'page');
+// Register media screens
+mediaScreenMap.registerByContentType(MediaSeries, CONTENT_TYPE.series);
+mediaScreenMap.registerByContentType(MediaSeriesEpisode, CONTENT_TYPE.episode);
+mediaScreenMap.registerByContentType(MediaLiveChannel, CONTENT_TYPE.livechannel);
+mediaScreenMap.registerByContentType(MediaStaticPage, CONTENT_TYPE.page);
 mediaScreenMap.registerDefault(MediaMovie);
 
-// register legacy series and episode screens when `contentType` is missing
+// Register legacy series and episode screens when `contentType` is missing
```

Comment:
So I noticed that ScreenMap.register is actually doing an `unshift` to add elements, which means that these legacy checks will take priority over the content types. We should probably change that to use a `push` so first in wins which is what developers are going to assume I think. Maybe that will also help us fix the situation with the legacy series flow that we were looking at on Friday.

Reply:
Looks like it won't let us fix the problem with a contentType as it is being used by some customers. However I think we can still use `push` for new logic to take precedence over the old flow.